src/components/CurrencyInputPanel/FiatValue.tsx

```diff
@@ -59,7 +61,7 @@ export function FiatValue({
             <span style={{ color: priceImpactColor }}>
               {' '}
               <MouseoverTooltip text={t`The estimated difference between the USD values of input and output amounts.`}>
-                (<Trans>{priceImpact.multiply(-1).toSignificant(3)}%</Trans>)
+                (<Trans>{formatPriceImpact(priceImpact)}%</Trans>)
```

Comment:
confirming this wont do a double %?

Reply:
it does indeed double it...my b ty for catching